graph/telemetry/istio/appender/aggregate_node_test.go

```diff
@@ -105,7 +105,7 @@ func TestNamespacesGraphWithServiceInjection(t *testing.T) {
 	assert.Equal(graph.NodeTypeAggregate, pp.Edges[1].Dest.NodeType)
 
 	topReviews := pp.Edges[0].Dest
-	if "Top" != topReviews.Metadata[graph.AggregateValue] {
+	if topReviews.Metadata[graph.AggregateValue] != "Top" {
```

Comment:
i don't understand the purpose of this churn. There's a few of these. These are identical statements in the end. I would like to see these gone.

Reply:
Done